numpy/_core/strings.py

```diff
@@ -12,29 +13,30 @@
     str_len, find, rfind, count, startswith, endswith,
     _lstrip_whitespace, _lstrip_chars, _rstrip_whitespace,
     _rstrip_chars, _strip_whitespace, _strip_chars,
+    _replace
 )
 
 
 __all__ = [
     "equal", "not_equal", "less", "less_equal", "greater", "greater_equal",
     "add", "isalpha", "isdigit", "isspace", "isdecimal", "isnumeric",
     "str_len", "find", "rfind", "count", "startswith", "endswith",
-    "lstrip", "rstrip", "strip",
+    "lstrip", "rstrip", "strip", "replace"
 ]
 
 
-def lstrip(x1, x2=None):
+def lstrip(a, chars=None):
     """
-    For each element in `x1`, return a copy with the leading characters
+    For each element in `a`, return a copy with the leading characters
     removed.
 
     Parameters
     ----------
-    x1 : array_like, with ``bytes_`` or ``unicode_`` dtype
-    x2 : scalar with the same dtype as ``x1``, optional
-       The ``x1`` argument is a string specifying the set of
-       characters to be removed. If ``None``, the ``x2``
-       argument defaults to removing whitespace. The ``x2`` argument
+    a : array_like, with ``bytes_`` or ``unicode_`` dtype
+    chars : scalar with the same dtype as ``a``, optional
+       The ``a`` argument is a string specifying the set of
```

Comment:
This was wrong before, but this should be `chars`, not `a`. Though I'd write it as
"A string specifying ..." since we're obviously talking about `chars` here...